src/ontology/mondo-ingest.Makefile

```diff
@@ -318,14 +318,10 @@ metadata/%-metrics.json: $(COMPONENTSDIR)/%.owl
 .PRECIOUS: metadata/%-metrics.json
 
 ../../docs/metrics/%.md: metadata/%-metrics.json | ../../docs/metrics/
-	j2 "$(SOURCE_METRICS_TEMPLATE)" metadata/$*-metrics.json > $@
-
-.PHONY: j2
-j2:
-	pip install j2cli j2cli[yaml]
```

Comment:
## Rationale for this change
So I guess that the `j2` Jinja CLI was never part of ODK, but `jinjanate` will be. So that is the reason we are going to start using `jinjanate`?
I assume it doesn't otherwise offer any special benefits?

Reply:
j2 was actually part of ODK for a while now (not when this was added to the makefile), but jinjanate is now part of the base packages. Already installed on `dev`.